organization/remuneration-policy.md

```diff
@@ -0,0 +1,38 @@
+---
+type: resource
+---
+
+# Remuneration policy
+
+This explains the Foundation for Public Code's approach to remuneration.
```

Comment:
I'd like to add a statement about our view on free labor on the open source community, and where we've specifically designed the Foundation to either encourage people to be paid for their work for the community or deliberately encourage volunteers.
(To address the issues from Ashe Dryden's post on OSS and unpaid labor: https://www.ashedryden.com/blog/the-ethics-of-unpaid-labor-and-the-oss-community)

Reply:
Great, seems like an awesome issue :)

Reply:
Done! #330